connector/elasticapmconnector/config.go

```diff
@@ -0,0 +1,282 @@
+		{Key: "k8s.pod.name"},
+		{Key: "service.version"},
+		{Key: "service.instance.id"},     // service.node.name
+		{Key: "process.runtime.name"},    // service.runtime.name
+		{Key: "process.runtime.version"}, // service.runtime.version
+		{Key: "telemetry.sdk.version"},   // service.language.version??
+		{Key: "host.name"},
+		{Key: "os.type"}, // host.os.platform
+		{Key: "faas.instance"},
+		{Key: "faas.name"},
+		{Key: "faas.version"},
+		{Key: "cloud.provider"},
+		{Key: "cloud.region"},
+		{Key: "cloud.availability_zone"},
+		{Key: "cloud.platform"}, // cloud.service.name
+		{Key: "cloud.account.id"},
+	}, serviceResourceAttributes...)
+
+	serviceSummaryAttributes := []signaltometricsconfig.Attribute{{
+		Key:          "metricset.name",
+		DefaultValue: "service_summary",
+	}}
+
+	serviceTransactionAttributes := []signaltometricsconfig.Attribute{
+		{Key: "transaction.root"},
+		{Key: "transaction.type"},
+		{Key: "metricset.name", DefaultValue: "service_transaction"},
+	}
+
+	transactionAttributes := []signaltometricsconfig.Attribute{
+		{Key: "transaction.root"},
+		{Key: "transaction.name"},
+		{Key: "transaction.type"},
+		{Key: "transaction.result"},
+		{Key: "event.outcome"},
+		{Key: "metricset.name", DefaultValue: "transaction"},
+	}
+
+	serviceDestinationAttributes := []signaltometricsconfig.Attribute{
+		{Key: "span.name"},
+		{Key: "event.outcome"},
+		{Key: "service.target.type"},
+		{Key: "service.target.name"},
+		{Key: "span.destination.service.resource"},
+		{Key: "metricset.name", DefaultValue: "service_destination"},
+	}
+
+	return &signaltometricsconfig.Config{
+		Logs: []signaltometricsconfig.MetricInfo{{
+			Name:                      "service_summary",
+			IncludeResourceAttributes: serviceResourceAttributes,
+			Attributes:                serviceSummaryAttributes,
+			Sum:                       &signaltometricsconfig.Sum{Value: "1"},
+		}},
+
+		Datapoints: []signaltometricsconfig.MetricInfo{{
+			Name:                      "service_summary",
+			IncludeResourceAttributes: serviceResourceAttributes,
+			Attributes:                serviceSummaryAttributes,
+			Sum:                       &signaltometricsconfig.Sum{Value: "1"},
+		}},
+
+		Spans: []signaltometricsconfig.MetricInfo{{
+			Name:                      "service_summary",
+			IncludeResourceAttributes: serviceResourceAttributes,
+			Attributes:                serviceSummaryAttributes,
+			Sum: &signaltometricsconfig.Sum{
+				Value: "Int(AdjustedCount())",
+			},
+		}, {
+			Name:                      "transaction.duration.histogram",
+			Description:               "APM service transaction aggregated metrics as histogram",
+			IncludeResourceAttributes: serviceResourceAttributes,
+			Attributes: append(serviceTransactionAttributes[:], signaltometricsconfig.Attribute{
+				Key:          "elasticsearch.mapping.hints",
+				DefaultValue: []any{"_doc_count"},
+			}),
+			Unit: "us",
+			ExponentialHistogram: &signaltometricsconfig.ExponentialHistogram{
```

Comment:
Lets default to histogram based buckets for now - Results looks promising and we need this in time for EDOT GA https://github.com/elastic/hosted-otel-collector/issues/406#issuecomment-2669574786 Thoughts?

Reply:
Done.